spec/unit/webmock_spec.rb

```diff
@@ -10,10 +10,10 @@
   end
 
   it "should alias enable_net_connect! to allow_net_connect!" do
-    expect(WebMock.enable_net_connect!).to eq(WebMock.allow_net_connect!)
+    expect(WebMock.method(:enable_net_connect!)).to eq(WebMock.method(:allow_net_connect!))
   end
 
   it "should alias disallow_net_connect! to disable_net_connect!" do
-    expect(WebMock.disallow_net_connect!).to eq(WebMock.disallow_net_connect!)
+    expect(WebMock.method(:disallow_net_connect!)).to eq(WebMock.method(:disallow_net_connect!))
```

Comment:
@owyongsk comprating disallow to disallow?

Reply:
Geez, I'm sorry. ):